mentoris/views.py

```diff
@@ -18,13 +18,18 @@ def sign_up(request):
     template = loader.get_template("mentapp/sign_up.html")
     return HttpResponse(template.render())
 
-  
+
 def profile(request):
-    template = loader.get_template('mentapp/profile.html')
+    template = loader.get_template("mentapp/profile.html")
     return HttpResponse(template.render())
 
+
 def user_info(request, user_id):
     user_profile = get_object_or_404(User, user_id=user_id)
-    email = get_object_or_404(Email, user_id=user_id)
-    return render(request, 'mentapp/profile.html', {'user_profile': user_profile,
-                                                    'email': email})
\ No newline at end of file
+    try:
+        email = Email.objects.get(user_id=user_id)
```

Comment:
You need to add a field for the email where `is_primary=True`. You can look at my previous PR or Noah's PR to see this. This is so that the correct email address gets retrieved.